charts/pega/README.md

```diff
@@ -1045,15 +1045,15 @@ Parameter   | Description   | Default value
-`hazelcast.enabled` |  Set as true if client-server deployment of Pega Platform is required, otherwise false. Note: Set this value as false for Pega platform versions below 8.6, if not set the installation will fail | `true`
-`hazelcast.clusteringServiceEnabled` |  Set as true if client-server deployment of Pega Platform is required, otherwise false. Note: Set this value as false for Pega platform versions below 8.8, if not set the installation will fail | `false`
-`hazelcast.migration.enabled` |  Set as true after creating parallel cluster to migrate the data and client switch over to new Hazelcast service | `false`
-`hazelcast.migration.migrationJobImage` | Reference the `platform/clustering-service-kubectl` Docker image that you downloaded and pushed to your Docker registry that your deployment can access. | `YOUR_MIGRATION_JOB_IMAGE:TAG`
-`hazelcast.migration.embeddedToCSMigration` |  Set as true while creating parallel cluster in case of embedded source system | `false`
-`hazelcast.migration.skipRestart` |  Set as true while removing old hazelcast cluster after migration to avoid restart | `false`
-`hazelcast.replicas` | Number of initial members to join the Hazelcast cluster | `3`
-`hazelcast.username` | UserName to be used in client-server Hazelcast model for authentication | `""`
-`hazelcast.password` | Password to be used in client-server Hazelcast model for authentication | `""`
+`hazelcast.enabled` |  Set to `true` if client-server deployment of Pega Platform is required, otherwise leave set to `false`. Note: To avoid an installation failure, you must set this value to `false` for Pega platform deployments using versions before 8.6. | `true`
+`hazelcast.clusteringServiceEnabled` |  Set to `true` if client-server deployment of Pega Platform is required, otherwise leave set to `false`. Note: Set this value to `false` for Pega platform versions below 8.8, if not set the installation will fail. | `false`
+`hazelcast.migration.enabled` |  Set to `true` after creating parallel cluster (new Hazelcast) to establish the connection with platform and migrate the data. | `false`
+`hazelcast.migration.migrationJobImage` | Reference the `platform/clustering-service-kubectl` Docker image to create the migration job to run the migration script. | `YOUR_MIGRATION_JOB_IMAGE:TAG`
+`hazelcast.migration.embeddedToCSMigration` |  Set to `true` while creating parallel cluster in case of embedded source system,  leave this value set to `false` in case of c/s to c/s migration. | `false`
+`hazelcast.migration.skipRestart` |  Set to `true` while removing old hazelcast cluster after migration to avoid pega pods restart | `false`
+`hazelcast.replicas` | Number of initial members to join the Hazelcast cluster. | `3`
+`hazelcast.username` | UserName to be used in client-server Hazelcast model for authentication. | `""`
```

Comment:
somewhere you should let clients know that the authentication is automatic when you set the values here. I do NOT think it is obvious.

Reply:
@taz-pega-work , username/password is existing field and being used by client since a long time. Added note as below: 
UserName to be used in client-server Hazelcast model for authentication; if not set the installation will fail.